packages/pixeloven/cli-addon-generators/package.json

```diff
@@ -37,6 +37,7 @@
         "lint:ts": "pixeloven-linter ts src/**/*.{ts,tsx}",
         "pretty": "pixeloven-pretty **/*.{json,md,ts,tsx}",
         "precommit": "lint-staged",
+        "precompile": "rm -rf dist",
```

Comment:
So we don't really want this because I added support for incremental builds and watch mode for ts. I think if anything we just add a robust cleanup step as you said.

Reply:
Oh yeah, I had to add this while I was experimenting because some files we didn't need were sticking around. I'll make sure to remove it.